### PR TITLE
[PHEE-477] Create an orb that fetches the docker images and do helm upgrade

### PR DESCRIPTION
**Pull request by logoutdhaval**
## Description

[PHEE-477 Create an orb that fetches the docker images and do helm upgrade](https://fynarfin.atlassian.net/browse/PHEE-477?atlOrigin=eyJpIjoiNDU5MjQzNzc4MGJhNDA1YmFhMmQ0MGI4OGU2ZmEyNDgiLCJwIjoiamlyYS1zbGFjay1pbnQifQ)

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Design related bullet points or design document link related to this PR added in the description above. 

- [ ] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

### CODE REVIEW

File: .circleci/docker-image-availability-check-and-upgrade.yml
Comment (fynmanoj) [inline .circleci/docker-image-availability-check-and-upgrade.yml:1]: Please create a documentation on how to use this ORB

Refer: https://circleci.com/developer/orbs/orb/circleci/helm

**Review by fynmanoj — CHANGES_REQUESTED** [review]
review 1

**Review by fynmanoj — CHANGES_REQUESTED** [review]

File: .circleci/docker-image-availability-check-and-upgrade.yml [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+            IMPORTER_ES: ph-ee-g2psandbox.ph-ee-engine.importer_es.image=docker.io/openmf/ph-ee-importer-es:latest
+            OPERATIONS_APP: ph-ee-g2psandbox.ph-ee-engine.operations_app.image=docker.io/openmf/ph-ee-operations-app:latest
+            OPERATIONS_WEB: ph-ee-g2psandbox.ph-ee-engine.operations_web.image=docker.io/openmf/ph-ee-operations-web:latest
+            VOUCHERS: ph-ee-g2psandbox.ph-ee-engine.vouchers.image=docker.io/openmf/ph-ee-vouchers:latest
+          command: |
+            if [ "$CIRCLE_BRANCH" != "master" ]; then
+            
+              PR_NUMBER=$(basename $CIRCLE_PULL_REQUEST)
+              PR_TITLE=$(curl -sSL "https://api.github.com/repos/openmf/ph-ee-env-template/pulls/$PR_NUMBER" | jq -r '.title')
+              JIRA_STORY=$(echo $PR_TITLE | cut -d "[" -f2 | cut -d "]" -f1)
```
[the remark this answers is not in the document]
Comment (logoutdhaval) [reply .circleci/docker-image-availability-check-and-upgrade.yml]: fixed

File: .circleci/docker-image-availability-check-and-upgrade.yml [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+              if [ -z "$JIRA_STORY" ]; then echo "Invalid PR title" && exit 1; else echo "Ticket NO: $JIRA_STORY"; fi
+              
+              function docker_tag_exists() {
+                curl --silent -f --head -lL https://hub.docker.com/v2/repositories/$1/tags/$2/ > /dev/null
+              }
+              
+              
+              if docker_tag_exists openmf/community-app ${JIRA_STORY}; then
```
[the remark this answers is not in the document]
Comment (logoutdhaval) [reply .circleci/docker-image-availability-check-and-upgrade.yml:82]: I can not use any loops here as env var are different for each service.

File: .circleci/docker-image-availability-check-and-upgrade.yml
Comment (logoutdhaval) [reply .circleci/docker-image-availability-check-and-upgrade.yml:1]: Here is the doc for creating and using an orb in circleci.

https://docs.google.com/document/d/1u-nhZVWaFqNXucdWn3RD50ObXdi6TEAwU-4lgYLyMAI/edit

**Review by fynmanoj — APPROVED** [review]

**Pull request merged**